primers.go

```diff
@@ -103,3 +104,79 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
+// DeBruijn generates a DNA DeBruijn sequence
+// https://rosettacode.org/wiki/De_Bruijn_sequences#Go
+// Pulled and adapted from here
+func DeBruijn(n int) string {
```

Comment:
@Koeng101 should this be a public function?

Reply:
Yes.

Reply:
Could you write a comment since it's exported so that it shows up in our docs?